src/components/ha-header-bar.ts

```diff
@@ -0,0 +1,66 @@
+import {
+  customElement,
+  LitElement,
+  html,
+  unsafeCSS,
+  css,
+  property,
+} from "lit-element";
+// @ts-ignore
+import topAppBarStyles from "@material/top-app-bar/dist/mdc.top-app-bar.min.css";
+
+@customElement("ha-header-bar")
+export class HaHeaderBar extends LitElement {
+  @property({ type: Boolean }) centerTitle = false;
+
+  protected render() {
+    let title = html`<span class="mdc-top-app-bar__title">
```

Comment:
Let's make this a ternary statement.

Reply:
or if…else.

Reply:
the second one uses the first one.

Reply:
But I ended up not using it because it doesn't work well, so might as well remove it...